client/reader/sidebar/reader-sidebar-recent/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { localize } from 'i18n-calypso';
+import { Component } from 'react';
+import ExpandableSidebarMenu from 'calypso/layout/sidebar/expandable';
+import ReaderFollowingIcon from 'calypso/reader/components/icons/following-icon';
+
+export class ReaderSidebarRecent extends Component< {
```

Comment:
Any particular reason why you went with a class component here instead of a functional one?

Reply:
I decided to follow the convention of the other expandable sidebar items to stub this out, figuring we could easily change to functional. It could be good to start with functional though, for pulling in the site data later. What's your recommendation?